packages/sdk/src/dataProtector/sharing/setProtectedDataToSubscription.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers, type Contract } from 'ethers';
+import { ABI as sharingABI } from '../../contracts/sharingAbi.js';
+import { WorkflowError } from '../../utils/errors.js';
+import { throwIfMissing } from '../../utils/validators.js';
+import {
+  AddressOrENS,
+  IExecConsumer,
+  SetProtectedDataToSubscriptionParams,
+  SetProtectedDataToSubscriptionResponse,
+} from '../types.js';
+
+export const setProtectedDataToSubscription = async ({
+  collectionTokenId = throwIfMissing(),
```

Comment:
@cedric25 i think `collectionTokenId` will work better as it is used in the smart contracts repo as well, it would be better if we have the same name across different packages.